src/assets/stylesheets/components/site/tables/_cell-inputs.scss

```diff
@@ -31,9 +31,11 @@
   > .table-cell-static-range {
     display: flex;
     flex-grow: 1;
+    align-items: flex-end;
     justify-content: space-evenly;
 
     > span {
+      max-width: 5ch;
```

Comment:
used character units to set a max width for the static range displays

Reply:
wow.  I honestly had no idea character units was even a thing.  noice.